packaging/packager_rpm.sh

```diff
@@ -7,8 +7,15 @@ readonly distro_root="${ROOT:-$HOME}"
 
 # rpm dependencies
 # python3
-dnf -y install python3 python3-coverage
+# dnf -y install python3 python3-coverage
+echo "Python installation"
+dnf -y install python3 
+echo "Python COVERAGE installation"
+dnf -y install python3-coverage
+#python version
+python3 --version
```

Comment:
I see, however I don't understand why the steps are separated now, they are more compact as a single step and coverage is the main python entry point used in these tests.